src/adapter/src/catalog/builtin.rs

```diff
@@ -2371,6 +2371,67 @@ JOIN mz_catalog.mz_roles role_owner ON role_owner.id = class_objects.owner_id
 WHERE mz_schemas.database_id IS NULL OR d.name = pg_catalog.current_database()",
 };
 
+pub const PG_DEPEND: BuiltinView = BuiltinView {
```

Comment:
We recently added `mz_object_dependencies` and `mz_object_transitive_dependencies`, is it possible to define this view in terms of those?

Reply:
Ah my mistake, I misread the definition.